keepassxc-browser/content/totp-field.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const ignoreRegex = /(bank|coupon|postal|user|zip).*code|comment|author|error/i;
-const ignoredTypes = [ 'email', 'password', 'username' ];
```

Comment:
I'm not sure why `password` was removed. Standard TOTP fields aren't `password` fields, so this should be kept.

Reply:
See #2332 solution one, sometimes TOTP fields *do* have `type=password`.
Another solution would be the second one (an explicit accept), then the `type!=password` would also be bypassed.

Reply:
Instead of making this change global, I'd suggest adding an exception for this site's TOTP detection to the `sites.js`.